validphys2/src/validphys/kinematics.py

```diff
@@ -96,6 +113,30 @@ def kinlimits(commondata, cuts, use_cuts, use_kinoverride: bool = True):
 
 all_kinlimits = collect(kinlimits, ('dataset_inputs',))
 
+def xq2_dataset_map(commondata, cuts,internal_multiclosure_dataset_loader,
+                        _internal_max_reps=None,
+                        _internal_min_reps=20):
+    """
+    Load in a dictionary all the specs of a dataset meaning:
```

Comment:
@giovannidecrescenzo, @andreab1997  I think that this stuff should stay in `multiclosure_inconsistent.py` or `multiclosure_inconsistent_output.py`

Reply:
With this stuff, I mean all of the single datapoint ratio bias variance analysis